backend/controllers/contributions.js

```diff
@@ -0,0 +1,27 @@
+const axios = require("axios")
+const config = require("../config/index")
+
+const AuthContribution = {
+  async verifyIdentity(token) {
+    return await axios.get(
+      `${config.netlifyContributionURL}/.netlify/identity/user`,
```

Comment:
weird to have a . in the url

Reply:
It's an API route given by Netlif,  I cannot change anything in the URL.